tests/test_benchmark.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from os import path
+from conftest import skipif
+from devito import configuration
+from subprocess import check_call
+
+
+pytestmark = skipif(['yask', 'ops'])
+
+
+@pytest.mark.parametrize('mode', ['bench'])
+@pytest.mark.parametrize('problem', ['acoustic', 'tti', 'elastic', 'viscoelastic'])
+def test_bench(mode, problem):
+    """
+    Test combinations of modes and problems.
```

Comment:
"Test the Devito benchmark framework on various combinations of modes and problems."